moto/codedeploy/models.py

```diff
@@ -492,5 +512,31 @@ def list_deployment_groups(
             for deployment_group in self.deployment_groups[application_name].values()
         ]
 
+    def list_tags_for_resource(
+        self, resource_arn: str
+    ) -> Dict[str, List[Dict[str, str]]]:
+        try:
+            return self.tagger.list_tags_for_resource(resource_arn)
+        except Exception:
+            return {"Tags": []}
```

Comment:
```suggestion
        return self.tagger.list_tags_for_resource(resource_arn)
```

IMO it would be better to fail loudly. If an error occurs, and an Exception is thrown for whatever reason, at least we know about it, and we can fix it. If we hide errors, it becomes much easier to figure out why Moto is not behaving like you'd expect.

Reply:
Updated!